src/api/app/models/event_subscription/find_for_event.rb

```diff
@@ -7,9 +7,55 @@ def initialize(event)
     end
 
     def subscriptions(channel = :instant_email)
+      if event.instance_of?(Event::RelationshipCreate) || event.instance_of?(Event::RelationshipDelete)
```

Comment:
I think this is not really needed. I'm gonna remove it

Reply:
so you would always use the `target_user_strategy` method right? If I see it correctly `receiver_roles_strategy` would not work with the `any_roles` role right?

Reply:
because the `any_role` is not implemented in `Event::Base`

Reply:
I should get rid of `target_user_strategy` and use only `receiver_roles_strategy`. We have `any_role` implemented in `Event::Relationship`